app/controllers/users_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+    return if @user
+
+    flash[:danger] = t "users.show.not_found"
+    redirect_to new_user_path
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t "users.create.success"
```

Comment:
Nos đang đúng path theo tổ chức i18n là user.create rồi thì e có thể viết ngắn gọn là
```suggestion
      flash[:success] = t ".success"
```